aria/modeling/service_instance.py

```diff
@@ -1632,9 +1673,9 @@ def interface_fk(cls):
         return relationship.foreign_key('interface', nullable=True)
 
     @declared_attr
-    def plugin_specification_fk(cls):
-        """For Operation one-to-one to PluginSpecification"""
-        return relationship.foreign_key('plugin_specification', nullable=True)
+    def plugin_fk(cls):
```

Comment:
does it make sense for operation no longer to have an fk for plugin specification at all?
is that going to remain for operation template?